dockers/app/Dockerfile

```diff
@@ -44,14 +51,21 @@ RUN mv /isard/isard.conf.docker /isard/isard.conf
 
 COPY dockers/app/certs.sh /
 COPY dockers/app/add-hypervisor.sh /
+COPY dockers/app/add-hyper-rethink.py /
 COPY dockers/app/supervisord.conf /etc/supervisord.conf
 
 EXPOSE 5000
 
 CMD ["/usr/bin/supervisord", "-c", "/etc/supervisord.conf"]
 
+
 FROM production as development
 RUN pip3 install --no-cache-dir ipython pytest
-RUN apk add --no-cache --update bash vim openssh bash
+RUN apk add --no-cache --update bash vim openssh 
+#py3-pexpect
```

Comment:
This comments could be deleted, since they're not used for anything

Reply:
ok